### PR TITLE
Quiz index

### PR DESCRIPTION
**Pull request by Wijantra**
I create quiz-index.html for display all of quiz , upvote and downvote for each quiz.

I add quiz-index.html, quiz-index.css, view for render quiz-index.html, quiz-index url and test for render quiz-index

image

### CODE REVIEW

File: quizer_game/templates/quizer_game/quiz-index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      			<!--<td>Dan</td>-->
+      			<!--<td>04.45</td>-->
+      			<!--<td>14</td>-->
+    		<!--</tr>-->
+		</tbody>
+	</table>
+</div>
+
+<div id="back">
+  <a href=>Back</a>
```
Comment (NutthanichN) [inline quizer_game/templates/quizer_game/quiz-index.html]: Should use 2 spaces for indentation.

Comment (Wijantra) [reply quizer_game/templates/quizer_game/quiz-index.html]: I already fix it.
Please check it

**Comment by NutthanichN** [comment]
I modify `quiz_index` view so it retrieves all quizzes from the database and displays them on `quiz-index.html`. I also add a button on `index.html` that links to `quiz-index.html`.

@Wijantra @theethaj Please review it.

**Comment by Wijantra** [comment]
I clone and check it, it run correctly

File: quizer_game/templates/quizer_game/quiz-index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <head> 
+    <meta charset="UTF-8">
+    <title>Quiz index</title>
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'quizer_game/styles/quiz-index.css' %}">
+  </head>
+  <body>
+
+    <header id="masthead">
+      login version
```
Comment (Wijantra) [inline quizer_game/templates/quizer_game/quiz-index.html]: I think you should hide it.

`login version`

Comment (NutthanichN) [reply quizer_game/templates/quizer_game/quiz-index.html]: Already fixed.

**Comment by NutthanichN** [comment]
> 
> 
> It works correctly but it also has errors according to Travis CI at quizer_game/templates. So, please check them before merging.

It's an old error, I already fixed it.

**Pull request merged**